src/AppStackNavigator.tsx

```diff
@@ -4,6 +4,7 @@ import { createNativeStackNavigator } from 'react-native-screens/native-stack';
 import ConfigurationNavigator from './Screens/Configuration/ConfigurationStackNavigator';
 import Home from './Screens/Home';
 import Login from './Screens/Login';
+
```

Comment:
Why did you commit this file? Is there any change?

Reply:
Probably some unintentional change that I made during the rebase. I will remove this file modification